File: 2556-ConvertTheTemperature/2556-ConvertTheTemperature.js
// Last updated: 3/6/2026, 2:08:37 PM
/**
 * @param {number} celsius
 * @return {number[]}
 */
var convertTemperature = function(celsius) {
     let Kelvin = celsius + 273.15;
    let Fahrenheit = celsius * 1.80 + 32.00;

    return [Kelvin,Fahrenheit];
};

File: 2556-ConvertTheTemperature/2556-ConvertTheTemperature.cpp
// Last updated: 7/17/2026, 4:24:36 PM
class Solution {
public:
    vector<double> convertTemperature(double celsius) {
        double kelvin = celsius + 273.15;
        double fahrenheit = celsius * 1.80 + 32.00;

        return {kelvin , fahrenheit};
    }
};